### ISSUE TITLE
Python dependency

### ISSUE THREAD

**Issue by vliedel** [overviewer/Minecraft-Overviewer#1956]
Hi, I tried installing minecraft overviewer on ubuntu 20.04 (which has python 3.8 by default), but that failed due to the python dependency:
```
$ apt depends minecraft-overviewer 
minecraft-overviewer
  Depends: python3 (<< 3.6)
  Depends: python3 (>= 3.5~)
  Depends: <python3:any> (>= 3.3.2-2~)
    python3:i386
    python3
  Depends: python3-pil
  Depends: python3-numpy
  Depends: libc6 (>= 2.4)
```

Is this because it doesn't work on python 3.8, or because you didn't test it yet?

**Comment by vliedel** [comment]
Building and running from source works fine for me at least.

**Comment by CounterPillow** [comment]
We need to add builds for each separate distro release, you cannot use a prebuilt version of a different python release because the python ABI is not stable across versions like that.

Overviewer will always work with the latest version of Python from source, because that is what I develop with, not whatever ancient release the Deboids ship.

Closed as duplicate of #1595.
